content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -71,6 +71,8 @@ For a reference to old feature gates that are removed, please refer to
 | `AnyVolumeDataSource` | `false` | Alpha | 1.18 | 1.23 |
 | `AnyVolumeDataSource` | `true` | Beta | 1.24 | |
 | `AppArmor` | `true` | Beta | 1.4 | |
+| `ClusterTrustBundle` | `false` | Alpha | 1.28 | |
```

Comment:
This is already in this file (see below on line 89).

Reply:
This is the key outstanding feedback blocking a merge, I think.

Reply:
Done.